vue-app/src/components/ConnectionModal.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="modal-body">
+    <div v-if="walletProvider">
```

Comment:
I think we solve should for the case that no `walletProvider` exists. Otherwise it pops the modal overlay with no content (& no way to remove the modal overlay).

Perhaps it's a simple modal that says: "No wallet found" with a button to cancel?

Reply:
Good catch. True, I haven't think about that case.

The "No wallet found" could be one possible way of dealing with it.

Later I'll check what we are doing in the Connect button in this case. Probably we could align this with that and display the same thing.